Source/Utils/RoomUtil.cs

```diff
@@ -94,6 +95,50 @@ private static object GetAula(HtmlNode? node)
         return new { name = nome, building = edificio, power = pwr, link = ROOM_INFO_URLS + info };
     }
 
+    public static async Task<object?> getRoomById(int id){
+        var url = ROOM_INFO_URLS + "Aula.do?" + 
+                  "idaula=" +  id;
+
+        var html = await HtmlUtil.DownloadHtmlAsync(url);
+        if (html.IsValid() == false) return null;
+        /*
+        example of property tag
+        <td colspan="1" rowspan="1" style="width: 33%" class="ElementInfoCard1 jaf-card-element">
+			<i>Codice vano</i>
+			<br>&nbsp;LCF040800S042
+		</td>
+        (parsing doesn't work very well, regex++)
+        */
+        string fetchedHtml = html.GetData() ?? "";
+        String[] fields = {"Sigla", "Capienza", "Edificio"};
+        //other fields include "Tipologia", "Indirizzo", "Dipartimento", "Codice vano", "Postazione per studenti disabili", ...
+        var properties = new Dictionary<String, String>();
+        int propLen = fields.Length;
+        for(int i = 0;i < propLen;i++){
+            String i_tag = $@"<i>{fields[i]}</i>";
+            Regex filter = new Regex($@"{i_tag}.*?<br>.*?</td>", RegexOptions.Singleline);
+            var match = filter.Match(fetchedHtml);
+            if(match.Success){
+                properties[fields[i]] = match.Value
+                    .Replace(i_tag, "")
+                    .Replace("<br>", "")
+                    .Replace("</td>", "")
+                    .Replace("&nbsp;", "")
+                    .Replace("\n", "").Trim();
+            }
+            else
+                properties[fields[i]] = "-";
+        }   
+        properties["Edificio"] = properties["Edificio"].Split('-')[0].Trim();
+        var json = File.ReadAllText("Other/Examples/roomsWithPower.json");
+        var data = JObject.Parse(json);
+        //Retrieving the list of IDs for the room with power outlets
+        var list = data["rwp"]?.Select(x => (int)x).ToArray();
+        if(list != null)
+            properties["prese"] = list.Contains(id) ? "si" : "no";

```

Comment:
```suggestion
            properties["prese"] = list.Contains(id) ? true : false
```
La risposta deve essere coerente con quella dell'endpoint delle aule libere, con un boolean qui e tutti i nomi delle proprietà in inglese (es. "Edificio", "Capienza", etc.)